### PR TITLE
MAIN - exports new version of encryption

### PR DESCRIPTION
**Pull request by paschalidi**
Hello, 

this will export the bits we decided to be publicly available. 

Please have a look and let me know.

### CODE REVIEW

File: src/tests/medical-sticker/charlie/index.contract.test.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ import {
   hash,
   splitKeys,
   fingerprint
-} from "../lib/MedStickerEncryptionCharlie";
+} from "../../../lib/medical-sticker/charlie";
```
Comment (lanwen) [inline src/tests/medical-sticker/charlie/index.contract.test.js]:
```suggestion
} from "../../../lib/medical-id/charlie";
```

**Comment by paschalidi** [comment]
@Weetbix just tried it in an app. 
after following your instructions, now I can import fucntions like this:
````
import { decrypt, hash } from "@vivy/krypt-web/dist-src/lib/medical-id/charlie";
````
this is the file structure that pika produces. 

Does it makes sense for us ?

**Comment by paschalidi** [comment]
NOTE: To test you can install the package like following npm i @vivy/krypt-web@{{local path of vivy-krypt-web-2.0.0.tgz}}

**Comment by Weetbix** [comment]
>import { decrypt, hash } from "@vivy/krypt-web/dist-src/lib/medical-id/charlie";

We discussed this in person. 

In summary, we don't want to use import paths like these because:
- it ties the consumers to our internal code layout
- it does not allow bundlers to pick the correct code target (as you must specify which folder exactly, rather than it looking at the package.json `esnext`, `module` and `main` fields. 

Seems the only way to import in this style: `import foo from "bar/module` is if that file is directly in the package root. But this also breaks the ability to have different modules/bundles and targets. 

So in the end we decide to go back to normal exports which can still be tree shaked and provide different bundle types, and will need to be imported like so:

```
import { medicalId } from "@vivy/krypt-web";
const { charlie : { encrypt } } = medicalId;
```

**Pull request merged**